RetailCoder.VBE/UI/Command/FindAllReferencesCommand.cs

```diff
@@ -87,7 +88,8 @@ private void UpdateTab()
 
         protected override bool EvaluateCanExecute(object parameter)
         {
-            if (_vbe.ActiveCodePane == null || _state.Status != ParserState.Ready)
+            if ((_state?.Status ?? ParserState.None) != ParserState.Ready ||
+                (_vbe.ActiveCodePane == null && !(_vbe.SelectedVBComponent?.HasDesigner ?? false)))
```

Comment:
If `_state` is `null`, we have one hell of a problem - it's safe to assume it's not null. That null-checking seems overkill.

Reply:
Fair enough, will remove. I added based on nullchecks I saw elsewhere.

Reply:
Let me know where they are if you encounter them - they should probably go as well :-)